projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -179,13 +179,21 @@ public function create_item( $request ) {
 		$plugins  = $request['plugins'];
 		usort( $plugins, 'strnatcasecmp' );
 
-		$event = Scheduled_Updates::create_scheduled_update( $schedule['timestamp'], $schedule['interval'], $plugins );
+		$res = Scheduled_Updates::create_scheduled_update( $schedule['timestamp'], $schedule['interval'], $plugins );
 
-		if ( is_wp_error( $event ) ) {
-			return $event;
+		if ( is_wp_error( $res ) ) {
+			// If the schedule could not be created, return an error.
+			$res->add_data( array( 'status' => 500 ) );
+
+			return $res;
 		}
 
-		$id = Scheduled_Updates::generate_schedule_id( $plugins );
+		$id     = Scheduled_Updates::generate_schedule_id( $plugins );
+		$events = wp_get_scheduled_events( Scheduled_Updates::PLUGIN_CRON_HOOK );
```

Comment:
We should be able to use `wp_get_scheduled_event()` here instead.

Reply:
Done.